mautic/mautic
# change string in company view

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | ✅ 
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | Not relevant
| Related user documentation PR URL      | 
| Related developer documentation PR URL | 
| Issue(s) addressed                     | 



#### Description:
I noticed that in the company view, the owner displayed as label Contact Owner while it should be written Company owner. This PR just modify the string to use the adapted one.

#### Steps to test this PR:

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create a company and assign an owner
3. View the page and see that it is written "Contact Owner" instead of "Company Owner", after applying the PR this is fixed.


<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/11288"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
